sources/components/custom-styles/template-cs-style-rules.scss

```diff
@@ -984,3 +989,128 @@ body {
+      .edy-product-widget-item-title,
+      .edy-product-widget-item-link {
+        color: var(--third-color);
+      }
+
+      .edy-product-widget-item-price,
+      .edy-product-widget-item-description {
+        color: var(--third-color);
+        opacity: .7;
+      }
+    }
+
+    .edy-product-widget-item-top-outer .edy-product-widget-item-top.edy-product-widget-item-without-image {
+      border: 1px solid var(--third-color);
+      opacity: .5;
+    }
+
+    .edy-product-widget-item-details-flex-wrap {
+      .edy-product-widget-item-out-of-stock,
+      .edy-product-widget-item-btn {
+        color: var(--third-color);
+      }
+    }
+    
+    &:hover {
+      .edy-product-widget-item-price {
+        opacity: 0;
+      }
+    }
+  }
+}
+
+.edy-product-widget-list .edy-product-widget-item-wrap .edy-product-widget-item .edy-product-widget-item-details-wrap {
+  .dark-background & {
+    .edy-product-widget-item-header {
+      .edy-product-widget-item-title {
+        .edy-product-widget-item-link {
+          color: var(--third-color);
+        }
+      }
+
+      .edy-product-widget-item-price {
+        color: var(--third-color);
+        opacity: .7;
+      }
+
+      .edy-product-widget-item-out-of-stock {
+        color: var(--third-color);
+      }
+    }
+    
+    .edy-product-widget-item-description {
+      color: var(--third-color);
+    }
+
+    .edy-product-widget-item-without-image {
+      border: 1px solid var(--third-color);
+      opacity: .5;
+    }
+  }
+
+  .edy-product-widget-item-btn-wrap .edy-product-widget-item-btn {
+    .light-background &,
+    .dark-background & {
+      padding: calc(var(--button-padding) - 18px) var(--button-padding) calc(var(--button-padding) - 17px);
+      font-size: var(--button-font-size);
+      font-style: var(--button-font-style);
+      font-weight: var(--button-font-weight);
+      hyphens: var(--button-hyphens);
+      text-decoration: var(--button-text-decoration);
+      text-transform: var(--button-text-transform);
+      background-color: transparent;
+      border: 2px solid var(--secondary-color);
+    }
+
+    .dark-background & {
+      border-color: var(--third-color);
+      color: var(--third-color);
+    }
+
+    .light-background & {
+      border-color: var(--secondary-color);
+      color: var(--secondary-color);
+    }
+
+    &:hover {
+      .dark-background & {
+        border-color: var(--third-color);
+        background-color: var(--third-color);
+        color: var(--primary-color);
+      }
+
+      .light-background & {
+        border-color: var(--secondary-color);
+        background-color: var(--secondary-color);
+        color: var(--third-color);
+      }
+    }
+  }
+}
+
+.dark-background {
+  .edy-product-widget-filters-item {
+
+    .edy-product-widget-filter-name,
+    .edy-product-widget-filter-sort,
+    .edy-product-widget-filter-label,
+    .edy-product-widget-filter-search-input {
+      color: white;
```

Comment:
There has problem with select BG in dark mode:
![Screenshot 2022-03-03 at 15 47 57](https://user-images.githubusercontent.com/378755/156577638-d025dad5-509f-4fe6-9426-b90a91b6e959.png)